Formula/fabio.rb

```diff
@@ -21,15 +19,14 @@ class Fabio < Formula
   depends_on "consul" => :recommended
 
   def install
-    mkdir_p buildpath/"src/github.com/ebay"
-    ln_s buildpath, buildpath/"src/github.com/ebay/fabio"
-
-    ENV["GOPATH"] = "#{buildpath}/_third_party:#{buildpath}"
+    mkdir_p buildpath/"src/github.com/eBay"
+    ln_s buildpath, buildpath/"src/github.com/eBay/fabio"
 
-    Language::Go.stage_deps resources, buildpath/"src"
+    ENV["GOPATH"] = buildpath.to_s
+    ENV["GO15VENDOREXPERIMENT"] = "1"
 
-    system "go", "build", "-o", "fabio"
-    bin.install "fabio"
+    system "go", "install", "-tags", "netgo", "github.com/eBay/fabio"
```

Comment:
This is trying to do the step of opening static libraries in the Go prefix. Can we disable that behaviour here?

```
==> go install -tags netgo github.com/eBay/fabio
go install net: open /usr/local/Cellar/go/1.6.2/libexec/pkg/darwin_amd64/net.a: operation not permitted
```